t/35-script_clone_job.t

```diff
@@ -17,10 +17,47 @@ use Test::Most;
 
 use FindBin;
 use lib "$FindBin::Bin/../lib";
+use Date::Format;
+use Test::Exception;
 use Test::Output 'combined_like';
 use OpenQA::Script::CloneJob;
 use Mojo::URL;
 use Mojo::File 'tempdir';
+use OpenQA::Test::Case;
+
+my $schema = OpenQA::Test::Case->new->init_data(skip_fixtures => 1);
```

Comment:
seems to help a bit but not really, 7.8s . I would prefer if t/35-script_clone_job.t focuses on anything without the database and at best also without a webapi server. You could extend t/09-job_clone.t though as that one already spawns a database.

Reply:
As discussed, I'm moving `09-job_clone` over to `api/04-jobs-api` along with the cloning-related tests from the "big" jobs API test in #3166 first, and we can avoid making 35-script_clone_job heavier.